pkg/trace/stats/weight.go

```diff
@@ -1,8 +1,7 @@
-package agent
+package stats
```

Comment:
Not sure if WeightedTrace/Span should be in the stats package in the end. Weight is not used at the agent level for any stats and is mostly used for sampling purposes (so I feel like a `traces` package might be a better spot?).

Reply:
Not sure either. I just put it there because the documentation mentioned that it is used in the concentrator. But happy to move it later. I'm a bit reticent about adding packages freely and would like to keep them to a minimum. I hope it's ok if we leave it here for now. If you have another existing package as an idea of where this would be better placed, I'm happy to move it in an immediate next PR.

I will merge this for now so I can continue the work.